File: src/typescript/src/transport/in-memory/__tests__/alpha-event.ts
import { messageInfo } from '../../transaction/message/message-info';
import { MessageType } from '../../transaction/message/message-type';

/**
 * Test event message for alpha endpoint
 */
@messageInfo(MessageType.Event, 'alpha', 'alpha-event', 1, 0, 0)
export class AlphaEvent {
  public name!: string;
}

File: src/kotlin/src/test/kotlin/polybus/transport/inmemory/AlphaEvent.kt
package polybus.transport.inmemory

import polybus.transport.transactions.messages.MessageInfo
import polybus.transport.transactions.messages.MessageType

@MessageInfo(MessageType.EVENT, "alpha", "alpha-event", 1, 0, 0)
class AlphaEvent {
    lateinit var name: String
}
